src/components/ChapterCard.tsx

```diff
@@ -22,27 +24,23 @@ interface Props extends WithStyles<typeof styles> {
     | subscribeChapterById_chapter;
 }
 
-const ChapterCard = ({ classes, chapter }: Props) => {
+const ChapterCard: React.FC<Props> = ({ classes, chapter }) => {
```

Comment:
What is the motivation for explicitly adding the FC type? (Just curious, as I am not fully up to date with TS and React. If I am not mistaken, it was not used in this project because of this: https://github.com/facebook/create-react-app/pull/8177.)

Reply:
TBH I've never thought this through, just a pattern I absorbed… 😅 Wasn't aware of the downsides. So the only advantage seems to be that TS yells at you if you don't return a component (or explicitly null), which isn't really necessary actually… I changed it back to stay consistent.

Reply:
Haha ok, well React with TypeScript are at times a bit confusing but yes this seems to be the take away. However it certainly doesn't matter here too much :)